static/new_password_required.html

```diff
@@ -0,0 +1,104 @@
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css"
+    />
+    <title>NEW_PASSWORD_REQUIRED</title>
+    <style>
+      .holder {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      .mt-s {
+        margin-top: 0.7em;
+      }
+      code {
+        font-size: 9px !important;
+        overflow-x: scroll;
+      }
+    </style>
+  </head>
+  <body class="holder">
+    <div class="ui raised text container">
+      <div class="column">
+        <h1 class="ui header">NEW_PASSWORD_REQUIRED</h1>
+        <div class="ui compact message hidden" id="message">
+          <p>Message goes here...</p>
+        </div>
+
+        <div class="ui styled fluid accordion">
+          <div class="active title">
+            <i class="dropdown icon"></i>
+            Username / Password
+          </div>
+          <div class="active content">
+            <form
+              class="ui form padded"
+              method="post"
+              action="/new_password_required"
+            >
+              <div class="field">
+                <label>Username</label>
+                <input
+                  type="text"
+                  name="username"
+                  id="username"
+                  readonly
+                  required
+                  placeholder="Please enter your username or refresh token"
+                />
+                <input type="hidden" name="session" id="session" required />
+              </div>
+              <div class="field">
+                <label>NewPassword</label>
+                <input
+                  type="password"
+                  name="password"
+                  required
+                  placeholder="Wh4t_i5_your_NEW_p@ssw0rd?"
+                />
+              </div>
+              <button class="ui button" type="submit">Submit</button>
+            </form>
+          </div>
+        </div>
+        <p class="mt-s">
+          <a href="/">← Back to index</a>
+        </p>
+
+        <code class="ui compact message hidden" id="params">
+          Message goes here...
+        </code>
+      </div>
+    </div>
+  </body>
+  <script
+    src="https://code.jquery.com/jquery-3.1.1.min.js"
+    integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+    crossorigin="anonymous"
+  ></script>
+  <script src="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.js"></script>
+  <script>
+    $(".ui.accordion").accordion();
+
+    if (document.location.search.includes("params")) {
+      try {
+        const m = document.getElementById("params");
+        m.style.display = "block";
+        const params = document.location.search.replace("?params=", "");
+        const str = decodeURIComponent(params);
+        const data = JSON.parse(str);
+        $("#username").val(data.username);
+        $("#session").val(data.session);
+      } catch (e) {
+        console.error(e);
+      }
```

Comment:
Based on my comment above for `login.go`, this would simplify to just reading params from URL, and `JSON.parse` wouldn't be needed.